src/app/public/modules/autocomplete/autocomplete.component.html

```diff
@@ -51,6 +49,17 @@
         {{ noResultsFoundText || ('skyux_autocomplete_no_results' | skyLibResources) }}
       </div>
     </div>
+    <div *ngIf="showAddButton"
+      class="sky-autocomplete-actions"
+    >
+      <button *ngIf="showAddButton"
+        class="sky-autocomplete-add sky-btn sky-btn-link"
+        type="button"
+        (click)="addButtonClicked()"
+      >
+        <sky-icon icon="add" iconType="skyux"></sky-icon>{{'skyux_autocomplete_add' | skyLibResources}}
```

Comment:
Is there a different icon needed for each theme?

Reply:
@Blackbaud-AdamFunderburk - thoughts on this one? Should we be using FontAwesome's add icon for default theme?

Reply:
Yes, default icons (FontAwesome) for default theme.

Reply:
Done